messaging/schema.py

```diff
@@ -277,11 +292,13 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         if recipient_selection := data.get("recipient_selection"):
             validate_recipient_selection_and_data(
                 recipient_selection,
-                data["event"] if "event" in data else message.event,
+                data.get("event", message.event),
```

Comment:
This is probably ok, just saying that now message.event is evaluated (Not returned, but evaluated) always regardless of whether `"event" in data` is true or not. The possible problem could be if evaluating `message.event` would cause a problem, but it's a ForeignKey in Message so probably it's ok, I think it'll return None if the ForeignKey is not set (blank=True at least in the model)—didn't check this though.